chef/data_bags/crowbar/bc-template-neutron.json

```diff
@@ -66,7 +67,7 @@
     "neutron": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 40,
+      "schema-revision": 41,
```

Comment:
Revision 41 was already taken by different pull request please take 42 (and rename 041_additional_external_networks.rb accordingly)